### ISSUE TITLE
Munger to echo labels in comments

### ISSUE THREAD

**Issue by timstclair** [kubernetes/contrib#1390]
Github doesn't include labels anywhere in email notifications, which can make it hard to prioritize Kubernetes issues by email.

Until github adds this feature, I propose we have the k8s-bot echo label additions or removals in a comment, in a format that's easily parsed by email filters.

Would this feature be welcome, or too spammy?

/cc @matchstick

**Comment by bgrant0607** [comment]
@timstclair Thanks for filing an issue. I totally agree. Was already on https://github.com/kubernetes/community/wiki/Roadmap:-Contributor-Experience

cc @kubernetes/contributor-experience

**Comment by davidopp** [comment]
> I propose we have the k8s-bot echo comment additions or removals in a comment

I assume you mean "echo label additions or removals"

I'm all in favor of this as long as we limit it to issues (not PRs) and do batching (often a slew of labels wlil be added to an issue at once -- a team or two, an area or two, a priority, etc., and getting one email for each will drive people insane). Also I'm not convinced "removed" is needed -- maybe just "added" is sufficient? I'm just worried about the spam potential, there is already a lot of bot spam.

**Comment by bgrant0607** [comment]
Just added is a good idea.

**Comment by lavalamp** [comment]
Every time we do this it costs API call quota. We should figure out how
many label set events we think this will observe and see if that's feasible.

**Comment by bgrant0607** [comment]
@lavalamp By quota, do you mean rate limits?

https://developer.github.com/v3/#rate-limiting

Are we hitting these limits now?

**Comment by bgrant0607** [comment]
And could we get N times the quota by creating N different bot accounts?

**Comment by lavalamp** [comment]
Yes, we're running close-ish to the limits.

We could get more quota by making more bot accounts. That would work right
up until github blocks us for abuse. (I asked github support what they
would think of this option.)

We can't get more quota. (I tried pretty hard.) If we rearchitected the
merge queue to use webhooks instead of polling them, then I could try
asking again but basically they're not going to be very sympathetic as long
as we're polling.
